p2p/protocol/identify/id_delta.go

```
@@ -15,6 +16,8 @@ const IDDelta = "/p2p/id/delta/1.0.0"
 
 // deltaHandler handles incoming delta updates from peers.
 func (ids *IDService) deltaHandler(s network.Stream) {
+	s.SetReadDeadline(time.Now().Add(StreamReadTimeout))
```

I'd prefer to explicitly ignore the error.

---

This is done.